common/browsered/field-validation-checks.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+// local dependencies
+const emailValidator = require('../middleware/email-tools')
+
+// Constants
+const NUMBERS_ONLY = new RegExp('^[0-9]+$')
+const MAX_AMOUNT = 10000000
+
+const validationErrors = {
+  required: 'This is field cannot be blank',
```

Comment:
Do we have story for i18n? Not for this PR, just if we have a placeholder for this to integrate it since we haven't done it jet for _nunjucks_

Reply:
Created story: PP-3082